### ISSUE TITLE
what is the expected epoch loss

### ISSUE THREAD

**Issue by psy2013GitHub** [justinpinkney/stable-diffusion#58]
i try to reproduce the result, but failed
here is my config:
gpu(v100): 2
batch size:2
accumulate_batches: 2

after training 500 epochs, the train epoch loss is about 0.01, but the generated image is bad.

can u guys paste your training metrics, such as epoch loss etc
i think this should help others a lot. thanks.

**Comment by hdparmar** [comment]
I'm kinda lost on the metrics as well.
Do you know what all this stands for?
I can guess the common ones like train/loss_epoch and stuff but what is?

train/loss_simple_epoch
train/loss_vlb_epoch
val/loss_simple
val/loss_vlb
val/loss_vlb_ema
val/loss_simple_ema
val/loss_ema?
